test/mock-settlement.js

```diff
@@ -1,19 +1,21 @@
+const { GBP } = require('../app/constants/currencies')
+const { AP } = require('../app/constants/ledgers')
+
 const FRN = require('./mock-components/mock-frn')
 const { FIVE_HUNDRED_POUNDS } = require('./mock-components/mock-value')
-const { GBP } = require('../app/constants/currencies')
-const SETTLEMENT_DATE = require('./mock-components/mock-settlement-date')
+const { STRING_FORMAT: SETTLEMENT_DATE } = require('./mock-components/mock-dates').SETTLEMENT
 const { SETTLEMENT_REFERENCE } = require('./mock-components/mock-settlement-reference')
 const { SFI_FIRST_PAYMENT: SFI_FIRST_PAYMENT_INVOICE_NUMBER } = require('./mock-components/mock-invoice-number')
 const { SOURCE_SYSTEM: SFI_SOURCE_SYSTEM } = require('./mock-components/mock-source-system')
-const TRUE = require('./mock-components/mock-settlement-status')
 
 module.exports = {
   sourceSystem: SFI_SOURCE_SYSTEM,
   invoiceNumber: SFI_FIRST_PAYMENT_INVOICE_NUMBER,
   frn: FRN,
   currency: GBP,
+  ledger: AP,
   value: FIVE_HUNDRED_POUNDS,
   settlementDate: SETTLEMENT_DATE,
   reference: SETTLEMENT_REFERENCE,
-  settled: TRUE
+  settled: true
```

Comment:
Why change it from constant?

Reply:
Was complete overkill -- do we need to create another `mock` file for each value that is `true`, i.e. `mock-completed-status`, `mock-started-status` (ik they're not bool) the alt being importing this _specific_ one and therefore the `mock-completed-status` and `mock-started-status` are assigned the value of a `mock-settlement` value which 1) doesn't make sense/align/isn't clear and 2) in this case if the status for `completed` isn't a bool but rather a ts (as it is) then this will cause issues

Reply:
You do not need to create mock-object for all instances, you just need to create mock-status with Constant TRUE and FALSE.

Reply:
please confirm if any action is required, not clear from above discussion

Reply:
Fair point @AbidemiAdio -- I dnt havva strong opinion on this @simonnineteen90